framework/test/unit/nag/consumption/nag-opensearch.test.ts

```diff
@@ -33,6 +33,13 @@ new OpenSearchCluster(stack, 'OpenSearch', {
 
 Aspects.of(stack).add(new AwsSolutionsChecks({ verbose: true }));
 
+NagSuppressions.addStackSuppressions(stack, [
+  {
+    id: 'CdkNagValidationFailure',
+    reason: 'Intended behavior',
```

Comment:
can you scope the suppression to the exact resource and justify? It's generally because of intrinsic functions